### PR TITLE
Er amends

### PR DESCRIPTION
**Pull request by elliotroyle**
PR initially for the first three actions within the #20 issue.

### CODE REVIEW

File: data/config_inputs/lhc_config_inputs_v5.xlsx
Comment (ASW-Analyst) [inline data/config_inputs/lhc_config_inputs_v5.xlsx:1]: Please can you remove the cell protection for cells `E22:I22` in each of the four `_incidental` tabs.

File: src/config/assumptions/assumptions_treatments_m3.R
Comment (ASW-Analyst) [inline src/config/assumptions/assumptions_treatments_m3.R:1]: Defining the `m3_inc_tub_yX` assumptions, TB rates, has been removed from the code but needs adding in.

**Review by ASW-Analyst — CHANGES_REQUESTED** [review]
Please see the amendments for the `lhc_config_inputs_v5.xlsx` user input spreadsheet and the [`src/config/assumptions/assumptions_treatments_m3.R`](https://github.com/NHS-Transformation-Unit/Lung_Health_Checks/blob/78e1497535573fdea9a6b24dcd95e2ec4ff01df6/src/config/assumptions/assumptions_treatments_m3.R#L260)

**Review by ASW-Analyst — COMMENTED** [review]
Amends to `lhc_config_inputs_v5.xlsx` and `src/config/assumptions/assumptions_treatments_m3.R` approved

**Comment by elliotroyle** [comment]
Added a package to the requirements (`stringr`) and adapted the `initial_treatment.R` function script and `y1_4_init_treatment.R` model script to adapt method for calculating incidentals. Will replicate this methodology for other Y1 scripts (M3, M12 and M24) before applying to Y2-5.

**Review by ASW-Analyst — COMMENTED** [review]
Amendments look good, please add in for the other periods and each model year.

**Comment by elliotroyle** [comment]
All changes made to replicate Y1 initial incidental methodology across M3, M12 and M24 in Y1 as well as all scripts in Y2-5.

**Review by ASW-Analyst — APPROVED** [review]
Amendments approved

**Pull request merged**